### ISSUE TITLE
BBB 2.3-dev recordings not processing after reboot

### ISSUE THREAD

**Issue by CamZie** [bigbluebutton/bigbluebutton#10846]
**Describe the bug**
After every reboot of our BBB v2.3-dev server the recordings are not being processed anymore. The `bbb-rap-starter.service` and `bbb-rap-resque-worker.service` were `inactive (dead)`.

It will only work after running `bbb-conf --restart` after every reboot.

I checked the status of the services with ` bbb-conf --status` and they are all active.

**Output of "bbb-record --watch" while recording**
```
--
0 timers listed.
--
● bbb-rap-starter.service - BigBlueButton recording processing starter
   Loaded: loaded (/usr/lib/systemd/system/bbb-rap-starter.service; disabled; vendor preset: enabled)
   Active: inactive (dead)

● bbb-rap-resque-worker.service - BigBlueButton resque worker for recordings
   Loaded: loaded (/usr/lib/systemd/system/bbb-rap-resque-worker.service; disabled; vendor preset: enabled)
   Active: inactive (dead)
--
```

**Comment by AK1912** [comment]
Did you solve this issue? i have exactly the same

**Comment by ffdixon** [comment]
Fixed in the upcoming 2.2-alpah5.

For now, you can fix this by enabling both services

```
  systemctl enable bbb-rap-starter.service
  systemctl enable bbb-rap-resque-worker.service
```
